tests/integration/kubernetes/confidential_common.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+# Copyright 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+SUPPORTED_HYPERVISORS=(
+#    "qemu-sev"
+    "qemu-snp"
+    "qemu-tdx"
+)
+
+function setup_unencrypted_confidential_pod() {
+    local IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+    local UNENCRYPTED_IMAGE_URL="${IMAGE_REPO}:unencrypted"
+    local TEST_DIR="$(mktemp -d /tmp/test-kata-confidential.XXXXXXXX)"
+    export SSH_KEY_FILE="${TEST_DIR}/container-ssh-key"
+
+    # Pull image
+    sudo docker pull "${UNENCRYPTED_IMAGE_URL}"
+    
+    # Get ssh key from docker image label and save to file
+    sudo docker inspect ${UNENCRYPTED_IMAGE_URL} \
```

Comment:
Are we against porting in these docker methods? They may obviously need to be updated per Tobin's remark regarding parsing the labels on an encrypted image.
https://github.com/kata-containers/tests/blob/4fec2b5554ea8c14992c82854776d91b8660860f/lib/common.bash#L565

Reply:
I'm wondering whether we actually have to inspect the image at all, or just have those as part of our repo and use them.

Reply:
I agree with @fidencio, having those files version'ed will be one less point of failure.

Let's put the SSH keys in `tests/confidential/fixtures`. See for example `fixtures/quay_verification/x86_64/` where we put GPG and signature files.

Reply:
I've created: https://github.com/kata-containers/kata-containers/pull/7596

This will make sure we'll be building the image ourselves, and we'll be using the public key we store in our repo.

It doesn't use fixtures yet, but I can adapt to that depending on what will make to main first, that one or the fixtures rewrite.

@UnmeshDeodhar, please, make sure you'll use that image as part of the tests.

Reply:
Changed the yaml to use new image.